chapter7/web-ui-example/pom.xml

```diff
@@ -107,7 +107,7 @@
 		</dependency>
 		<dependency>
 			<groupId>org.seleniumhq.selenium</groupId>
-			<artifactId>selenium-firefox-driver</artifactId>
+			<artifactId>selenium-chrome-driver</artifactId>
```

Comment:
Unfortunately the book talks specifically about Firefox in chapter 7 page 215. We cannot change this. The pom files should do what they did before.